zip.go

```diff
@@ -93,10 +93,11 @@ func zipFile(w *zip.Writer, source string) error {
 
 		if baseDir != "" {
 			name, err := filepath.Rel(source, fpath)
+			name = strings.Replace(name, string(os.PathSeparator), "/", -1)
 			if err != nil {
```

Comment:
Also, there should be everything you need to manipulate paths in [`path/filepath`](https://godoc.org/path/filepath) and [`path`](https://godoc.org/path) packages. It shouldn't be neccessary to use `strings.Replace`.